lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_release_backmerge_pull_request_action.rb

```diff
@@ -0,0 +1,163 @@
+require 'fastlane/action'
+require_relative '../../helper/github_helper'
+
+module Fastlane
+  module Actions
+    class CreateReleaseBackmergePullRequestAction < Action
+      DEFAULT_BRANCH = 'trunk'.freeze
+
+      def self.run(params)
+        token = params[:github_token]
+        repository = params[:repository]
+        release_branch = params[:release_branch]
+        default_branch = params[:default_branch]
+        target_branches_param = params[:target_branches]
+        labels = params[:labels]
+        milestone_title = params[:milestone_title]
+
+        unless release_branch.start_with?('release/')
+          UI.user_error!('`release_branch` must start with `release/`')
+        end
+
+        if target_branches_param.include?(release_branch)
+          UI.user_error!('`target_branches` must not contain `release_branch`')
+        end
+
+        github_helper = Fastlane::Helper::GithubHelper.new(github_token: params[:github_token])
+        target_milestone = milestone_title.nil? ? nil : github_helper.get_milestone(repository, milestone_title)
+
+        target_branches = if target_branches_param.empty?
+                            determine_target_branches(release_branch.delete('release/'), default_branch)
+                          else
+                            target_branches_param
+                          end
+
+        pr_urls = []
+
+        target_branches.each do |target_branch|
+          Fastlane::Helper::GitHelper.checkout_and_pull(release_branch)
+
+          pr_urls << create_backmerge_pr(
+            token: token,
+            repository: params[:repository],
+            title: "Merge #{release_branch} into #{target_branch}",
+            head_branch: release_branch,
+            base_branch: target_branch,
+            milestone: target_milestone&.number,
+            labels: labels
+          )
+        end
+
+        pr_urls
+      end
+
+      def self.determine_target_branches(release_version, default_branch)
+        release_branches = Actions.sh('git branch -r -l "origin/release/*"').chomp
```

Comment:
I'd suggest to use the more secure approach of passing each individual command parameter as a separate value.

```suggestion
        release_branches = Actions.sh('git', 'branch', '-r', '-l', 'origin/release/*').chomp
```
The difference is that if you pass a single `String`, this will invoke a shell, which will interpret that string to split it into a command and its argument. That parsing of the command line by the shell not only is unnecessary in our case, but also opens the door to command injections, issues with spaces in arguments, or unwanted interpolations.
While if you pass each argument separately, it doesn't need to invoke a shell at all and can directly call the `git` program/process passing it each argument (already provided as separate ones) as its `argv` list, without involving any parsing of the command line to split it in arguments first.